slobrok/src/tests/mirrorapi/mirrorapi.cpp

```diff
@@ -140,10 +136,9 @@ Test::Main()
     cloud::config::SlobroksConfig::Slobrok slobrok;
     slobrok.connectionspec = "tcp/localhost:18501";
     specBuilder.slobrok.push_back(slobrok);
-    FRT_Supervisor orb;
-    MirrorAPI mirror(orb, config::ConfigUri::createFromInstance(specBuilder));
+    fnet::frt::StandaloneFRT server;
+    MirrorAPI mirror(server.supervisor(), config::ConfigUri::createFromInstance(specBuilder));
```

Comment:
the supervisor auto-start will probably make this test unstable (mirror will get ready), consider using explicit transport with late start.

Reply:
Fixed, but is this only a test issue, or is it also a race condition with construction of the MirrorAPI ?